pkg/server/sandbox_stop.go

```diff
@@ -65,7 +65,11 @@ func (c *criService) StopPodSandbox(ctx context.Context, r *runtime.StopPodSandb
 	state := sandbox.Status.Get().State
 	if state == sandboxstore.StateReady || state == sandboxstore.StateUnknown {
 		if err := c.stopSandboxContainer(ctx, sandbox); err != nil {
-			return nil, errors.Wrapf(err, "failed to stop sandbox container %q in %q state", id, state)
+			stateName := "Ready"
```

Comment:
suggest adding a String() function for sandbox.Status.State
See cri api for example:
```
type PodSandboxState int32

const (
	PodSandboxState_SANDBOX_READY    PodSandboxState = 0
	PodSandboxState_SANDBOX_NOTREADY PodSandboxState = 1
)

var PodSandboxState_name = map[int32]string{
	0: "SANDBOX_READY",
	1: "SANDBOX_NOTREADY",
}

var PodSandboxState_value = map[string]int32{
	"SANDBOX_READY":    0,
	"SANDBOX_NOTREADY": 1,
}

func (x PodSandboxState) String() string {
	return proto.EnumName(PodSandboxState_name, int32(x))
}
```
If cri wasn't missing the unknown pod state I'd say use that.. but since it's missing we should just create our own I suppose. Or you could do a switch .. either way maybe we should use the name format ^ cri is using.

Reply:
Great idea! I updated the PR. See if this is what you had in mind.

Just to be clear, what names do you prefer for the states?  I just used the enum names exactly: StateReady, StateNotReady, and StateUnknown, but can use all caps if you prefer. In the example you posted it looks like it left off the prefix part, but the namig convention is a bit odd.

Reply:
Forgot to gofmt, pushed again.

Reply:
I reread your comment... it sounds like you like the all upper case names SANDBOX_READY and SANDBOX_NOTREADY, and add one called SANDBOX_UNKNOWN?

Reply:
yes.. let's use their existing naming convention to reduce confusion and give them something to google :-)